src/util/file-filter.js

```diff
@@ -99,7 +96,7 @@ export class FileFilter {
    * file in the folder that is being archived.
    */
   wantFile(filePath: string): boolean {
-    const resolvedPath = this.resolve(filePath);
+    const resolvedPath = path.resolve(filePath);
```

Comment:
Can you explain why this change is needed? If I put it back to `this.resolve()` then all tests still pass.